js/Dot.js

```diff
@@ -1148,12 +1226,16 @@ Track.prototype.updateSelected = function(selected) {
 Track.prototype.style_schema = function() {
 	var styles = [
 		{name: "Annotations", type: "section"},
-		{name: "test", type: "bool", default: false},	
+		{name: "arrowToggled", type: "bool", default: false},
```

Comment:
Could you call arrowToggled something more user-friendly like "show as arrows"? Another good option is to use the dropdown selector, which has the bonus of supporting multiple types later if necessary -- see "alignment symbol" in DotPlot's styles.

Reply:
That's a good idea.